loopme-sdk/src/main/java/com/loopme/request/RequestBuilder.java

```diff
@@ -188,6 +189,7 @@ public static JSONObject buildRequestJson(Context context, LoopMeAd loopMeAd) th
                     .put(TIMEZONE, requestUtils.getTimeZone())
                     .put(ORIENTATION, requestUtils.getOrientation())
                     .put(CHARGE_LEVEL, requestUtils.getChargeLevel(context))
+                    .put(BATTERY_SAVER, requestUtils.getBatterySaverState(context))
```

Comment:
in charge level you need to put INT value depend of battery % 
<img width="114" alt="Screenshot 2024-08-19 at 18 03 42" src="https://github.com/user-attachments/assets/87c0de56-a7eb-4767-ba42-43326f948a5d">

Reply:
fixed in new commit, thanks

Reply:
fixed in new commit